packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -149,7 +149,7 @@ export function useAsyncData<
     nuxt._asyncData[key] = {
       data: ref(getCachedData() ?? options.default!()),
       pending: ref(!hasCachedData()),
-      error: toRef(nuxt.payload._errors, key),
+      error: ref(nuxt.payload._errors[key] ?? null),
```

Comment:
this will lose reactivity/sync state. I think we likely need to keep using `toRef` but initialise it beforehand